polaris.shopify.com/scripts/gen-cache-json.mjs

```diff
@@ -73,26 +75,32 @@ const getMdContent = (filePath) => {
   return {frontMatter: data, slug};
 };
 
-const genCacheJson = () => {
-  if (!existsSync(cacheDir)) mkdirSync(cacheDir, {recursive: true});
+const genCacheJson = async () => {
+  const spinner = ora(
+    'Generating .cache/nav.json and .cache/site.json',
+  ).start();
+
```

Comment:
should wrap in try / catch with spinner.fail() for any potential uncaught errors that might occur in this fn.